Sources/VLCEditController.swift

```diff
@@ -18,37 +18,36 @@ class VLCEditController: NSObject {
     private let collectionView: UICollectionView
     private let category: MediaLibraryBaseModel
 
-//    private lazy var editToolbar: VLCEditToolbar = {
-//        let editToolbar = VLCEditToolbar(frame: CGRect(x: 0, y: 400,
-//                                                       width: collectionView.frame.width, height: 50))
-//        editToolbar.isHidden = true
-//        editToolbar.delegate = self
-//        return editToolbar
-//    }()
+    private lazy var editToolbar: VLCEditToolbar = {
```

Comment:
Out of curiosity, shouldn't we leave this commented out since we don't yet have a toolbar?

Reply:
yeah can do